.gitignore

```diff
@@ -161,4 +161,7 @@ cython_debug/
 
 .vscode/
 .benchmarks/
-.DS_Store
\ No newline at end of file
+.DS_Store
+
+agentops_time_travel.json
+agentops_time_travel.yaml
```

Comment:
Can use use `.agentops_time_travel.json` and `.agentops_time_travel.yaml` as the file names? The . signifies that the file should be hidden (in Unix environments), which I think should be the case here. Let me know if users should be looking at these files as then, the above comment does not apply

Reply:
.agentops_time_travel.yaml can be hidden
agentops_time_travel.json isn't pretty but I think everyone said we should allow users to edit the cache offline without going through the dashboard